### ISSUE TITLE
[1.0.7] executePackageRequest caught an exception

### ISSUE THREAD

**Issue by alexandrius** [vscode-contrib/vscode-versionlens#254]
VersionLens fails to check versions after 1.0.7 (1.0.6 works fine) update:

```
error npm: executePackageRequest caught an exception.
 Package: {"name":"react-native","version":"https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz","path":"/Users/alexanderpataridze/Documents/Development/Toptal/stillwhite-app"}
 Error: {"source":"remote","data":"Not implemented yet"}
```

here's the package.json file:
https://pastebin.com/gsQ74VDn

thanks

**Comment by pflannery** [comment]
> VersionLens fails to check versions after 1.0.7 (1.0.6 works fine) update:
> 
> ```
> error npm: executePackageRequest caught an exception.
>  Package: {"name":"react-native","version":"https://github.com/expo/react-native/archive/sdk-37.0.1.tar.gz","path":"/Users/alexanderpataridze/Documents/Development/Toptal/stillwhite-app"}
>  Error: {"source":"remote","data":"Not implemented yet"}
> ```
> 
> here's the package.json file:
> https://pastebin.com/gsQ74VDn
> 
> thanks

I found why. 


The version of pacote that is currently being used (9.5.12) throws this error. 
Pacote had to be rolled back in 1.0.7 because the newer version didnt support auth.

Version lens will need to handle this error gracefully but can't move pacote forward yet.

**Comment by pflannery** [comment]
The fix for this is now released in [1.0.8](https://github.com/vscode-contrib/vscode-versionlens/blob/master/CHANGELOG.md#108)
